### PR TITLE
Docs index

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

**Comment by keszybz** [comment]
I don't know if the last part will render properly.  But we can't test this before merging.

File: man/systemd.scope.xml [hunk 1]
```diff
@@ -46,14 +46,14 @@
...
 
-    <para>Note that unlike service units scope units have no "main" process, all processes in the scope are
-    equivalent. The lifecycle of the scope unit is thus not bound to the lifetime of one specific process but
-    to the existance of any processes in the scope. This also means that the exit status of these processes
-    do not cause the scope unit to enter a failure state. Scope units may still enter a failure state, for
-    example due to resource exhaustion or stop timeouts being reached, but not due to programs inside of them
-    terminating uncleanly. Since processes managed as scope units generally remain children of the original
-    process that forked them off it's also the job of that process to collect their exit statuses and act on
-    them as needed.</para>
+    <para>Note that unlike service units, scope units have no "main" process, i.e. all processes in the scope
```
Comment (dnicolodi) [inline man/systemd.scope.xml]: I think `Note that, unlike service units, scope units have no "main" process: all processes in the scope are equivalent.` reads better.

File: docs/index.md [hunk 2]
```diff
@@ -24,3 +22,8 @@ See the introductory blog story and three status updates for a longer introducti
   {% endif %}
 {% endfor %}
 
+## See also
+
+* [Introductory blog story](https://0pointer.de/blog/projects/systemd.html)
```
Comment (dnicolodi) [inline docs/index.md]: The `https` version of this link does not work, at least from here. The `http` version does.

**Comment by keszybz** [comment]
Thanks for the review, updated.

**Comment by poettering** [comment]
Hmm, the "screenshot" didn't work out as it was supposed to

**Comment by keszybz** [comment]
> Hmm, the "screenshot" didn't work out as it was supposed to

It's not bad... just misplaced. Will fix.

**Pull request merged**